### ISSUE TITLE
PHPDOC: Unusal ending of comment

### ISSUE THREAD

**Issue by rugk** [atom/language-php#116]
This is your PHPDOC ([`f_doc`](https://github.com/atom/language-php/blob/d0f4aca74877c932dd9090421ac6448afd497def/snippets/language-php.cson#L145-L147)) snippet:
```php
/**
 * undocumented function summary
 *
 * Undocumented function long description
 *
 * @param type var Description
 **/
public function FunctionName($value='')
{
    # code...
}
```

However the comment ending as `**/` is quite unusual. Usually (and as defined by [phpdoc](http://phpdoc.org/docs/latest/glossary.html#term-docblocks)) you just end it with `*/`.

It would be nice if you could correct this.

----

Originally wrongly reported in https://github.com/jonrohan/docs-snippets/issues/10

**Comment by itiut** [comment]
This issue is fixed by #121, isn't it?

**Comment by 50Wliu** [comment]
Yup, thanks for the heads-up @itiut!
